nestedbox_core.entity.inc

```diff
@@ -77,6 +77,7 @@ function nestedbox_core_entity_info() {
       'label' => 'label',
     ),
     'access callback' => 'nestedbox_type_access',
+    'static cache' => TRUE,
```

Comment:
According to the doc comment on hook_entity_info(), this is TRUE by default. So no need to put it here.. right?